app/config/campaigns.yml

```diff
@@ -53,7 +53,7 @@ campaigns:
       description: Test different iterations of address pages
       reference: "https://phabricator.wikimedia.org/T366576"
       start: "2024-07-01"
-      end: "2024-12-31"
+      end: "2030-12-31"
```

Comment:
Giving it another 5 years makes this more of a feature than an A/B test. I'd like to have a discussion with the PM if there are really "tests" going on around this, that might conclude at some point (where we will remove the `<FeatureToggle>` tags and the duplicated address forms from the frontend) or if they consider the 2 address forms now a "feature" of the fundraising application. In the latter case we'd need a bit of refactoring to make the fundraising application act on the `ap` parameter without the A/B test campaigns. This is to avoid "feature toggle creep" and give the feature explicitly named code for better readability and visibility instead of having to guess and look up what the feature toogle ID means and getting pointed to 2023/2024 banner tickets